docs/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1201,6 +1201,11 @@ The archive may include any number of other files,
 which are accessible in the build context (See the [*ADD build
 command*](../../reference/builder.md#add)).
 
+Before the Docker daemon runs the instructions in the `Dockerfile`, it performs
+a preliminary validation of the `Dockerfile` and returns error if the syntax is
+incorrect. After that, each instruction is run one-by-one until the ID of the new
+image is output.
```

Comment:
Was thinking;

```Markdown
The Docker daemon performs a preliminary validation of the `Dockerfile` before
starting the build, and returns an error if the syntax is incorrect. After that,
each instruction is run one-by-one until the ID of the new image is output.
```

Perhaps `preliminary` should be `cursory`?

@duglin @mstanleyjones could you have a look at my suggestions?